index.txt

```diff
@@ -39,16 +39,16 @@ install_caddy()
 
 	# Termux on Android has $PREFIX set which already ends with /usr
 	if [[ -n "$ANDROID_ROOT" && -n "$PREFIX" ]]; then
-		install_path="${PREFIX}/bin"
+		install_path="$PREFIX/bin"
 	fi
 
 	# Fall back to /usr/bin if necessary
-	if [ ! -d "${install_path}" ]; then
+	if [[ ! -d $install_path ]]; then
 		install_path="/usr/bin"
 	fi
 
 	# Not every platform has or needs sudo (see issue #40)
-	[[ $EUID -ne 0 && -z "$ANDROID_ROOT" ]] && sudo_cmd="sudo"
+	[[ ((EUID)) && -z "$ANDROID_ROOT" ]] && sudo_cmd="sudo"
```

Comment:
The double parenthesis is evaluating arithmetic expressions. Since EUID is always numeric, it is more straightforward and easier to read. ((X)) is equivalent to ((X!=0)) as 0 is taken to evaluate to false here and non-zero to true. For extra readability you could use ((EUID != 0)) here, but it boils down to the same. It is a very common bash pattern, and often used in "boolean" expressions, like: ((can_mount)) && mount